door-open-service/node-backend/src/StatsDatabase.js

```diff
@@ -45,41 +49,52 @@ DoorOpens.sync();
 class StatsDatabase {
 
   static registerDoorOpen(user) {
```

Comment:
If you prefer the style, you can destructure the user object on the parameter list: `static registerDoorOpen({ name, email }) {`.